components/brave_new_tab_ui/components/default/braveNews/cards/CardImage.tsx

```diff
@@ -16,27 +15,28 @@ type Props = {
 }
 
 export default function CardImage(props: Props) {
-  const unpaddedUrl = useUnpaddedImageUrl(props.imageUrl, props.onLoaded)
+  const imageUrl = `chrome://maybe-padded-image?url=${props.imageUrl}`
```

Comment:
chrome://image takes some parameters, so we should call `encodeURIComponent` on the input URLs to prevent them from injecting parameters. see https://github.com/brave/brave-browser/issues/43367.

Reply:
Whoops thanks - got this confused with `chrome://favicon` which breaks if you use `encodeURIComponent`